web/app/tournament/tourny_helpers.py

```diff
@@ -0,0 +1,79 @@
+from flask import current_app, render_template, redirect, url_for
+from flask import jsonify, request, Response, flash
+from app.models import db, Tournament, TournamentPlayer, Match, Round
+
+def empty_field(form_data, fetched_db_data):
```

Comment:
This method seems a bit unnecessary or at least can be shortened.

Could be just  not (form_data or fetched_db_data) 

```python
def is_empty(a, b): 
  return not (a or b)
```

or it could be made a variadic function!

```python
def are_all_empty(*args):
  for count, thing in enumerate(args):
    if thing: 
      return false 
  return true
```